### ISSUE TITLE
Implement MP-BGP

### ISSUE THREAD

**Issue by pl0xym0r** [k01ek/netbox-bgp#184]
MP-BGP was initially planned but not implemented

 models session:    afi_safi = None  # for future use

+ choices: class AFISAFIChoices(ChoiceSet):

**Describe the solution you'd like**
Implement MG BGP on the plugin

**Describe alternatives you've considered**
What is done on nautobot https://github.com/nautobot/nautobot-app-bgp-models/blob/develop/nautobot_bgp_models/models.py
but not sure it could cover all needs on MP-BGP

@k01ek what was in your mind for this feature at first?

**Comment by movedempackets** [comment]
Hi @pl0xym0r,

I too would like an AFI-SAFI/address families field on session. Can I suggest that the field is also included on peer group.

**Comment by ajackson79** [comment]
I agree there needs to be AFI-SAFI support for tracking things like redistribution, network statements, aggregate routes, multipath, add path, etc. Also need support for VRFs.

**Comment by pl0xym0r** [comment]
@ajackson79 For VRF support, please consider #127

**Comment by ajackson79** [comment]
> @ajackson79 For VRF support, please consider #127

You need the vrf for a neighbor, but you also need it for the AFI/SAFI so you can track things like redistribution, network statements, aggregate routes, etc. Just adding vrf support to a neighbor is not enough to document everything.

**Comment by k01ek** [comment]
I think AFI/SAFI should be just a choice field and it should include in unique_together, it will allow to create session with same src/dst/as/ip but with different afi/safi. As for VRF, my thought is same, we don't need to create new field to link VRF, becasue local IP address has this link.
